packages/auth/src/index.ts

```diff
@@ -103,8 +103,7 @@ export function initIocContainer(
   })
 
   container.singleton('openApi', async (deps) => {
-    const config = await deps.use('config')
-    const clientSpec = await createOpenAPI(config.authServerSpec)
+    const clientSpec = await createOpenAPI('./openapi/auth-server.yaml')
```

Comment:
What if we rename the fetched AS spec to match the existing local specs:
```suggestion
    const clientSpec = await createOpenAPI('./openapi/client.yaml')
```

Reply:
I feel compelled to use the original name since it's being pulled in from somewhere external. The association with that spec's externality seems stronger that way, and it's holding the actual GNAP implementation so I feel auth-server might be a better name anyways.